src/Entity/Projection/ReferentManagedUser.php

```diff
@@ -173,6 +173,11 @@ class ReferentManagedUser
      */
     private $createdAt;
 
+    /**
```

Comment:
To be consistent with this class, you can add this PHPDoc : 
`* @var string|null`